packages/ui/components/pages/Fuse/Modals/Terms.tsx

```diff
@@ -0,0 +1,168 @@
+import {
+    Button,
+    UnorderedList,
+    ListItem,
+    List,
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalFooter,
+    ModalBody,
+    ModalCloseButton,
+    Text,
+  } from '@chakra-ui/react'
+
+  import React from 'react';
+  import { useState, useEffect } from 'react';
+
+
+  const Terms = () => {
+
+    const [hasAcceptedTerms, setHasAcceptedTerms] = useState<boolean>(Boolean(document.cookie.split('; ').find((row) => row.startsWith('MidasTandCAccepted='))?.split('=')[1]));
+
+    let isOpen = !hasAcceptedTerms;
+    const accept = () => {
+      document.cookie = 'MidasTandCAccepted=true;';
+      setHasAcceptedTerms(true);
+    }
+
+    //styling for Terms Sections and Sub-sections
+    const h1 = {as: 'u', fontSize: 'xl', fontWeight: 'bold'};
+    const h2 = {as: 'u', fontSize: 'l', fontWeight: 'normal'};
+
+    return (
+        <Modal isOpen={isOpen} closeOnOverlayClick={false} onClose={accept} scrollBehavior='inside' size={'6xl'}  >
+          <ModalOverlay />
+          <ModalContent>
+            <ModalHeader>Terms & Conditions</ModalHeader>
+            <ModalBody>
+              <Text as={h1.as} fontSize={h1.fontSize} fontWeight={h1.fontWeight}>  1. Acceptance of Terms </Text>
```

Comment:
If you like to use `<h1>` keep in mind that every single html page should only have a single `<h1>` element for accessibility reasons, screen readers, search engines etc.

So we should make use of all heading elements `<h1>,<h2>,<h3>,<h4>...`

best practice at least.

Reply:
Just saw you defined `h1` yourself as an object on top. 
Was confused by the name. It works sure.

Chakra ui, the component framework we are using offers `Heading` and `Text` components. Better make sure to use those here.

Why should the elemnt be a `u` instead of a `p` which is probably used by `Text`?